### PR TITLE
Out of the box clarification

### PR DESCRIPTION
**Pull request by lu-zero**
To clarify the out-of-the-box section I moved up the terminology section since few terms from there are used there and I added the, hopefully, shortest explanation of it.

***
<a href="https://pr-preview.s3.amazonaws.com/luminem/wot-profile/pull/415.html" title="Last updated on Jun 19, 2024, 6:48 PM UTC (52e6d93)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-profile/415/f68e08a...luminem:52e6d93.html" title="Last updated on Jun 19, 2024, 6:48 PM UTC (52e6d93)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -243,32 +292,36 @@ <h2>Motivation</h2>
...
-        for Profiles were derived from these use cases. 
+        for Profiles were derived from these use cases.
       </p>
     </section>
 
     <section id="out-of-the-box-interoperability">
       <h2>Out-of-the-Box Interoperability</h2>
+      <p>
+        In layman terms out-of-box-interoperability implies that every capability of a conformant
+        device can be perused by a conformant client, within the scope of the profile.
```
[the remark this answers is not in the document]
Comment (egekorkan) [reply index.html:320]: Also I am not sure peruse is the correct term here. I would just use "use" or activating a capability?

**Review by egekorkan — CHANGES_REQUESTED** [review]
Also, I think that terminology is always after introduction in W3C documents. In TD and Arch, it is like that.

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (lu-zero) [reply index.html:320]: Yes, you are right. (folding and rebasing)

**Comment by egekorkan** [comment]
> If we use the terms in parts of the introduction the terminology has to come first.

I agree. It is generally the case in all introductions but terminology still comes later, for some reason I do not know. My only guess is to have an introduction without making the user scroll a long time to get there, which can happen if the terminology is long.

File: index.html [hunk 2]
```diff
@@ -243,32 +292,36 @@ <h2>Motivation</h2>
...
+        for Profiles were derived from these use cases.
       </p>
     </section>
 
     <section id="out-of-the-box-interoperability">
       <h2>Out-of-the-Box Interoperability</h2>
+      <p>
+        In layman terms, out-of-box-interoperability implies that every capability of a conformant
+        Thing can be used by a conformant Consumer, within the scope of the profile.
+      </p>
```
Comment (benfrancis) [inline index.html:321]: I'm not sure these are really layman's terms 😆, and the definitions of "profile" and "out-of-the-box interoperability" would depend on each other.

How about...

```suggestion
      <p>
        At a high level, out-of-the-box interoperability means that a Consumer is guaranteed to be able to
        use every capability of a Thing, without Thing-specific customization.
      </p>
```

File: index.html [hunk 3]
```diff
@@ -243,32 +292,36 @@ <h2>Motivation</h2>
...
       <p>
         The definition of out-of-the-box interoperability used in this
-        specification includes multiple layers. 
+        specification includes multiple layers.
```
Comment (benfrancis) [inline index.html:324]: With the addition of the shorter high level definition above, we might want to make this say "The _full_ definition of out-of-the-box interoperability used in this specification includes multiple layers."

File: index.html [hunk 4]
```diff
@@ -171,69 +171,118 @@
...
     <p>
-      Future versions of this specification, or extension specifications, may 
+      Future versions of this specification, or extension specifications, may
       define additional Profiles.
     </p>
   </section>
 
   <section id='sotd'></section>
 
+  <section id="terminology">
```
Comment (benfrancis) [inline index.html:181]: We should probably leave this where it is for consistency with the other WoT specifications, but could link the word "profile" in the introduction to its definition in the terminology section for clarity.

**Review by benfrancis — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (lu-zero) [reply index.html:181]: I opened the discussion about it during today's main call. Cross linking is indeed a better solution.

**Comment by lu-zero** [comment]
Approved as per today TF call.
